integrated_channels/xapi/statements/learner_course_enrollment.py

```diff
@@ -16,21 +16,19 @@ class LearnerCourseEnrollmentStatement(EnterpriseStatement):
     xAPI statement to serialize data related to course registration.
     """
 
-    def __init__(self, user, course_overview, user_details, course_details, *args, **kwargs):
+    def __init__(self, user, user_social_auth, course_overview, *args, **kwargs):
         """
         Initialize and populate statement with learner info and course info.
 
         Arguments:
             user (User): Auth User object containing information about the learner enrolling in the course.
+            user_social_auth (UserSocialAuth): UserSocialAuth object for learner
```

Comment:
Not sure how much it matters, but this comment is redundant and doesn't really help to explain what the argument actually represents, it just restates the name of the argument.